backend/app/exceptions.py

```diff
@@ -254,3 +254,19 @@ class InvalidMultisigSignatureRequest(OctantException):
 
     def __init__(self):
         super().__init__(self.description, self.code)
+
+
+class InvalidMultisigAddress(OctantException):
```

Comment:
remove?

Reply:
I guess this is showed as a difference due to the fact that my base branch is ```[feature/oct-1541-rewards-calculator-update](https://github.com/golemfoundation/octant/tree/feature/oct-1541-rewards-calculator-update)``` not develop. And the mentioned base branch isn't aligned with develop yet so this class is highlighted here (but it's already on the develop).

@jmikolajczyk can you merge current state of develop with your branch as to keep the consistency?